Lib/multiprocessing/__init__.py

```diff
@@ -19,9 +19,8 @@
 # Copy stuff from default context
 #
 
-globals().update((name, getattr(context._default_context, name))
-                 for name in context._default_context.__all__)
-__all__ = context._default_context.__all__
+__all__ = [x for x in dir(context._default_context) if not x[0].startswith('_')]
```

Comment:
Shouldn't `x[0].startswith('_')` be `x.startswith('_')`?

Reply:
Yes, you are right. Thank you🙂